src/chrome/content/rules/dronebl.org.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="DroneBL">
+        <target host="dronebl.org" />
+        <target host="www.dronebl.org" />
```

Comment:
Please also add:
```
eu1.dronebl.org
frontend1.dronebl.org
mirror1.dronebl.org
```
and a comment about the non-working host `lists.dronebl.org`.

Our code style uses tabs to indent lines.

Reply:
`lists.dronebl.org` -> NXDOMAIN.

Reply:
`frontend1.dronebl.org` is also NXDOMAIN.